src/Look/InfoBoxLook.cpp

```diff
@@ -60,14 +60,19 @@ InfoBoxLook::Initialise(bool _inverse, bool use_colors,
 }
 
 void
-InfoBoxLook::ReinitialiseLayout(unsigned width)
+InfoBoxLook::ReinitialiseLayout(unsigned width, unsigned scale_title_font)
 {
-  const unsigned max_font_height = Layout::FontScale(12);
+  // const unsigned max_font_height = Layout::FontScale(12);
 
   FontDescription title_font_d(8);
-  AutoSizeFont(title_font_d, width, _T("0123456789"));
+  AutoSizeFont(title_font_d,
+    (width * scale_title_font) / 100U,
+    _T("123456789012345"));
```

Comment:
I think you can limit this to 14Chars. This will slightly increase the title size, and it even in current default you wouldn't see more than 14 Chars.